src/docs/guides/rocket.md

```diff
@@ -0,0 +1,188 @@
+
+### Deploy from the CLI
+
+1. **Install the Railway CLI**:
+    - <a href="/guides/cli#installing-the-cli" target="_blank">Install the CLI</a> and <a href="/guides/cli#authenticating-with-the-cli" target="_blank">authenticate it</a> using your Railway account.
+2. **Initialize a Railway Project**:
+    - Run the command below in your Rocket app directory. 
+        ```bash
+        railway init
+        ```
+    - Follow the prompts to name your project.
+    - After the project is created, click the provided link to view it in your browser.
+3. **Deploy the Application**:
+    - Use the command below to deploy your app:
+        ```bash
+        railway up
+        ```
+    - This command will scan, compress and upload your app's files to Railway. You’ll see real-time deployment logs in your terminal.
+4. **Set Up a Public URL**:
+    - Navigate to the **Networking** section under the [Settings](/overview/the-basics#service-settings) tab of your new service.
+    - Click [Generate Domain](/guides/public-networking#railway-provided-domain) to create a public URL for your app.
+
+    **Note:** You'll come across a 502 error where your application doesn't respond. We'll fix that in the next step.
+5. **Configure Rocket app to accept non-local connections**:
+    - Rocket apps need to be configured to accept external connections by listening on the correct address, which is typically `0.0.0.0`. You can easily do this by setting the address through the environment variable.
+    Run the following command to set the Rocket address to `0.0.0.0`:
+        ```bash
+        railway variables --set "ROCKET_ADDRESS=0.0.0.0"
+        ```
+6. **Redeploy the Service**:
+    - Run `railway up` again to trigger a redeployment of the service.
+7. **Verify the Deployment**:
+    - Once the deployment completes, go to **View logs** to check if the server is running successfully. Access your public URL again and you should see your app working well.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/f_auto,q_auto/v1729858389/docs/quick-start/rocket_app_service.png"
+alt="screenshot of the deployed Rocket service"
+layout="responsive"
+width={2038} height={1698} quality={100} />
+
+### Deploy from a GitHub Repo
+
+To deploy a Rocket app to Railway directly from GitHub, follow the steps below:
+
+1. **Create a New Project on Railway**:
+    - Go to <a href="https://railway.app/new" target="_blank">Railway</a> to create a new project.
+2. **Deploy from GitHub**: 
+    - Select **Deploy from GitHub repo** and choose your repository.
+        - If your Railway account isn’t linked to GitHub yet, you’ll be prompted to do so.
+3. **Add Environment Variables**:
+    - Click **Add Variables**, then add **ROCKET_ADDRESS** with the value `0.0.0.0`. This allows your Rocket app to accept external connections by listening on `0.0.0.0`.
```

Comment:
IMO `ROCKET_ADDRESS` should be in a code block.

Reply:
True